gfx/map/map_object_data/generated/steppe_bush_01_generator.txt

```diff
@@ -5,7 +5,7 @@
 	generated_content=yes
```

Comment:
Blackwald doesn't blend well with the rest of Gilneas.
It looks like a black hole.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132694628-64b96e7a-b402-40a7-9e00-91908960c2c3.png)

</details>

Reply:
_I'll reduce the intensity on the colormap in the next commit. Perhaps in the future when Gilneas gets a darker coloration (I certainly think it needs one!), the Blackwald can be made darker again._

Reply:
Should be even more blended. If you look at this place, you may notice red isn't a dominant color here. It's just the color of grass here and there with gray veins of the default Gilneas color.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132878494-ac0f98bb-c6f9-4237-8adc-f299d7ae2501.png)

</details>

Reply:
By the way, if you need it, you can resize colormap.

Reply:
Like, make it 2 times bigger.